tests/testTriggers.go

```diff
@@ -4548,6 +4552,92 @@ func TriggerPoolAddDisk(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	updateMetrics(*event)
 }
 
+// TriggerPoolAutoDisk performs auto-disk on the storage pools for the given contexts
+func TriggerPoolAutoDisk(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
```

Comment:
add triggerInterval also for these events

Reply:
Done.